docs/dnsupdate.rst

```diff
@@ -382,7 +382,7 @@ via the domainmetadata table:
 
 This will:
 
-1. Add the 'dhcpdupdate' key to our PowerDNSinstallation
+1. Add the 'dhcpdupdate' key to our PowerDNS installation
```

Comment:
This fix is because @Habbie spotted the typo while reviewing the expect file changes in this PR